src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -5,20 +5,27 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
-using System.Linq;
 using System.Reflection;
 using System.Text;
 using Microsoft.CodeAnalysis.Sarif.Driver;
-using Microsoft.CodeAnalysis.Sarif.Visitors;
-using Microsoft.CodeAnalysis.Sarif.WorkItemFiling;
+using Microsoft.CodeAnalysis.Sarif.WorkItems;
 using Microsoft.Json.Schema;
 using Microsoft.Json.Schema.Validation;
-using Microsoft.WorkItemFiling;
-using Newtonsoft.Json;
+using Microsoft.WorkItems;
 
 namespace Microsoft.CodeAnalysis.Sarif.Multitool
 {
-    public class FileWorkItemsCommand : CommandBase
+    /// <summary>
+    /// A class that drives SARIF work item filing. This class is responsible for 
+    /// collecting and verifying all options relevant to driving the work item filing
+    /// process. These options may be retrieved from a serialized version of the 
+    /// aggregated configuration (currently rendered as XML, via the PropertiesDictionary
+    /// class). Command-line arguments should override any options specified in the 
```

Comment:
>should [](start = 39, length = 6)

Remove "should". You're describing the implementation, which presumably actually does this.

Reply:
Indeed.

---
In reply to: [381009210](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381009210) [](ancestors = 381009210)